python/py-fmf/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        teemtee fmf 1.4.1
+
```

Comment:
please add "github.tarball_from archive" or if upstream provides releases on GitHub use "releases" instead

Reply:
sorry, I should have added that this change will require you to update the checksums. The port now fails because of the invalid ones...